packages-common/pluggable-widgets-generator/generators/app/lib/text.js

```diff
@@ -0,0 +1,48 @@
+const chalk = require("chalk");
+
+module.exports = {
+    getBanner: pkg =>
+        [
```

Comment:
I think you can use backticks and have a multiline string

Reply:
Changed it.